static/css/artist/artist_list.css

```diff
@@ -70,3 +73,35 @@
 .nav-float {
     float: left;
 }
+
+.artist-name {
+    padding-top: 5px;
+    padding-bottom: 0px;
+    font-family: Dosis, sans-serif;
+    background-color: #ecf0f1;
+}
+
+.artist-location {
+    padding-top: 5px;
+    padding-bottom: 10px;
+    font-family: Dosis, sans-serif;
```

Comment:
There seem to be a lot of shared attributes between `artist-location`, `artist-invested`, and `artist-earned`. Maybe there should be a class with the common attributes that's used in the same place?